### PR TITLE
Improve invitation list

### PR DESCRIPTION
**Pull request by jesusantguerrero**
Fix delete invites

Add resend invite

Add copy invitehash

### CODE REVIEW

**Comment by jesusantguerrero** [comment]
Note: Ask for resend invite endpoint

**Review by SparoHawk — COMMENTED** [review]
Two comments more than anything:
1. We could use a copy package to avoid having compatibility issues. What do you think?
2. I do not know why we need to have a copy invite link. Personally I think it should always be that you re-send the invitation email. Copying the link could promote having users with incorrect emails in the system.

**Comment by jesusantguerrero** [comment]
1- Under the hood, the libraries use the same pattern I used and as this was the only case I didn't want to add a library just for this case. But if we had/will have other use cases when this method is not enough we should go with a library.

2- The use case that max said if we want to send the link through chat for faster but I agree, re-send the invitation is more secure. I will talk to him.

**Comment by jesusantguerrero** [comment]
Update: we are going to use just the resend here and delete the copy functionality

**Comment by SparoHawk** [comment]
1.- Understandable. Let's do that then.
2.- Awesome. Let me know when the changes are pushed.

**Comment by jesusantguerrero** [comment]
Done

File: src/components/companies/users/list.vue [hunk 1]
```diff
@@ -171,6 +187,27 @@ export default {
...
+            }).catch((error) => {
+                this.$notify({
+                    group: null,
+                    title: "Error",
+                    text: error.response.data.errors.message,
+                    type: "error"
+                });
+            })
+        }, 
+
```
Comment (SparoHawk) [inline src/components/companies/users/list.vue]:
```suggestion
```

File: src/components/companies/users/list.vue [hunk 2]
```diff
@@ -171,6 +187,27 @@ export default {
...
         },
+        sendInvite(usersId) {
+            axios({
+                url: `/${this.endpoint}/${usersId}/resend`,
+                method: "POST"
+            }).then(() => {
+                this.$notify({
+                    group: null,
+                    title: "Invitation sent",
+                    text: "The invite has been sent",
```
Comment (SparoHawk) [inline src/components/companies/users/list.vue]:
```suggestion
                    text: "A new invitation link has been sent to the user.",
```

**Review by SparoHawk — CHANGES_REQUESTED** [review]

**Review by SparoHawk — APPROVED** [review]

**Pull request merged**